.circleci/config.yml

```diff
@@ -25,13 +29,6 @@ commands:
             INTERNAL_ADDR=localhost:<< parameters.port >> \
             WEBSOCKET_ADDR=localhost:<< parameters.port >> \
             ./test/healthcheck.sh
-      - run:
```

Comment:
Could you please keep this, I plan to include this new config into the 1.0 (legacy) tag?

Reply:
done